### PR TITLE
Api 2338 font 404 fix

### PR DESCRIPTION
**Pull request by bdgrider**
[font_404_change_summary.docx](https://github.com/department-of-veterans-affairs/vets-saml-proxy/files/5285478/font_404_change_summary.docx)

Updated static assets with latest versions from formation (fonts, img, css). Made it so that only one route serves statics assets. Fixed all font 404's and removed temporary patch code from previous font 404.

### CODE REVIEW

**Comment by beaugrantham-va** [comment]
I'm still seeing a few 404s on the error page.

```
saml-proxy_1  | {
saml-proxy_1  |   "remote-address": "::ffff:172.18.0.1",
saml-proxy_1  |   "time": "2020-09-26T19:14:46.355Z",
saml-proxy_1  |   "method": "GET",
saml-proxy_1  |   "url": "/samlproxy/webfonts/fa-solid-900.woff",
saml-proxy_1  |   "status-code": "404",
saml-proxy_1  |   "content-length": "1011",
saml-proxy_1  |   "referrer": "http://localhost:7000/samlproxy/idp/core.css",
saml-proxy_1  |   "request_id": "8a4857d0-002c-11eb-82a2-2b0b10e56b49"
saml-proxy_1  | }
```

File: saml-proxy/public/formation.min.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...arge-block-grid-5>li:nth-of-type(1n){clear:none}.large-block-grid-5>li:nth-of-type(5n+1){clear:both}.large-block-grid-6>li{list-style:none;width:16.66667%}.large-block-grid-6>li:nth-of-type(1n){clear:none}.large-block-grid-6>li:nth-of-type(6n+1){clear:both}.large-block-grid-7>li{list-style:none;width:14.28571%}.large-block-grid-7>li:nth-of-type(1n){clear:none}.large-block-grid-7>li:nth-of-type(7n+1){clear:both}.large-block-grid-8>li{list-style:none;width:12.5%}.large-block-grid-8>li:nth-of-type(1n){clear:none}.large-block-grid-8>li:nth-of-type(8n+1){clear:both}.large-block-grid-9>li{list-style:none;width:11.11111%}.large-block-grid-9>li:nth-of-type(1n){clear:none}.large-block-grid-9>li:nth-of-type(9n+1){clear:both}.large-block-grid-10>li{list-style:none;width:10%}.large-block-grid-10>li:nth-of-type(1n){clear:none}.large-block-grid-10>li:nth-of-type(10n+1){clear:both}.large-block-grid-11>li{list-style:none;width:9.09091%}.large-block-grid-11>li:nth-of-type(1n){clear:none}.large-block-grid-11>li:nth-of-type(11n+1){clear:both}.large-block-grid-12>li{list-style:none;width:8.33333%}.large-block-grid-12>li:nth-of-type(1n){clear:none}.large-block-grid-12>li:nth-of-type(12n+1){clear:both}}
```
Comment (beaugrantham-va) [inline saml-proxy/public/formation.min.css:1]: Is this file necessary? I don't see it actually being referenced anywhere.

File: saml-proxy/styles/core.scss [hunk 2]
```diff
@@ -1,21 +1,15 @@
 $font-path: "/samlproxy/idp/fonts";
+$formation-asset-path: "../idp";
```
Comment (beaugrantham-va) [inline saml-proxy/styles/core.scss:2]: I recommend replacing this with an absolute path (like above).

**Review by beaugrantham-va — CHANGES_REQUESTED** [review]

**Comment by crolarlibertyva** [comment]
Local testing in my environment shows that no 404's  are occurring for fonts.

File: saml-proxy/public/formation.min.css (hunk 1 above)
Comment (bdgrider) [reply saml-proxy/public/formation.min.css:1]: It was recommended in the formation install instructions. Removing it does not appear to have any effect on  the test procedures I went through, so ill go ahead and remove it.

**Review by beaugrantham-va — APPROVED** [review]

**Pull request merged**